machine.go

```diff
@@ -487,6 +487,8 @@ func (m *Machine) startVMM(ctx context.Context) error {
 	m.logger.Printf("Called startVMM(), setting up a VMM on %s", m.Cfg.SocketPath)
 	startCmd := m.cmd.Start
 
+	m.logger.Debug("[startVMM] arguments:", m.cmd.Args)
```

Comment:
The `[startVMM]` prefix is not aligned with other log messages. How about `Debugf("Starting %v", m.cmd.Args)`

Reply:
Updated the log call to the suggested format with 653e0ea